tests/types/SDK.Wwise.txt

```diff
@@ -10,3 +10,15 @@ DeadByDaylight/Plugins/Wwise/ThirdParty/x64_vc160/Release/bin/AkSoundEngineDLL.d
 EtG_Data/Plugins/AkSoundEngine.dll
 Firewatch.app/Contents/Plugins/AkSoundEngine.bundle
 YookaLaylee64_Data/Plugins/AkSoundEngine.dll
+WwiseGDNative.dll
+/WwiseGDNative.dll
+libWwiseGDNative.so
+/libWwiseGDNative.so
+/libwwise.windows.template_release.dll
+/libwwise.windows.template_debug.dll
+libwwise.windows.template_release.dll
+libwwise.windows.template_debug.dll
+/libwwise.linux.template_release.so
+/libwwise.linux.template_debug.so
+libwwise.linux.template_release.so
+libwwise.linux.template_debug.so
```

Comment:
```suggestion
WwiseGDNative.dll
/WwiseGDNative.dll
build/windows/WwiseGDNative.dll
```

Reply:
Why not include Linux libraries as well?

Reply:
> 

The projects include Linux versions, so why not account for those as well?